packages/wonder-blocks-modal/generated-snapshot.test.js

```diff
@@ -710,6 +713,82 @@ describe("wonder-blocks-modal", () => {
         expect(tree).toMatchSnapshot();
     });
     it("example 9", () => {
+        const {StyleSheet, css} = require("aphrodite");
+        const Button = require("@khanacademy/wonder-blocks-button").default;
+        const {View} = require("@khanacademy/wonder-blocks-core");
+        const {Strut} = require("@khanacademy/wonder-blocks-layout");
+        const Spacing = require("@khanacademy/wonder-blocks-spacing").default;
+        const {
+            HeadingLarge,
+            Body,
+        } = require("@khanacademy/wonder-blocks-typography");
+
+        const styles = StyleSheet.create({
+            previewSizer: {
+                height: 512,
+            },
+
+            modalPositioner: {
+                // Checkerboard background
+                backgroundImage:
+                    "linear-gradient(45deg, #ccc 25%, transparent 25%), linear-gradient(-45deg, #ccc 25%, transparent 25%), linear-gradient(45deg, transparent 75%, #ccc 75%), linear-gradient(-45deg, transparent 75%, #ccc 75%)",
+                backgroundSize: "20px 20px",
+                backgroundPosition: "0 0, 0 10px, 10px -10px, -10px 0px",
+
+                display: "flex",
+                flexDirection: "row",
+                alignItems: "center",
+                justifyContent: "center",
+
+                position: "absolute",
+                left: 0,
+                right: 0,
+                top: 0,
+                bottom: 0,
+            },
+
+            imageContainer: {
+                position: "absolute",
+                width: "100%",
+                height: "100%",
+                backgroundImage: "url('pencilHand-800x.png')",
+                backgroundSize: "contain",
+                backgroundRepeat: "no-repeat",
+                backgroundPosition: "center bottom",
```

Comment:
Can you change this to use an `<img>` tag to show that `fullBleedSidebar` is working?

Offtopic: `fullBleedSidebar` would make an awesome name for a metal band.

Reply:
I added an example with `<img>` but also kept the original to show that a background image is good for circumstances where the image is purely background thing and shouldn't be dragged around.

Reply:
`fullBleedSidebar` 🤘